enterprise/frontend/src/embedding-sdk/hooks/private/setup-sdk-auth.unit.spec.tsx

```diff
@@ -9,18 +8,17 @@ import api from "metabase/lib/api";
 
 const JWT_CONFIG = createMockJwtConfig();
 const API_KEY_CONFIG = createMockApiKeyConfig();
-const NO_CONFIG = {};
 
 const setup = ({ config }: { config: SDKConfig }) => {
   const dispatch = jest.fn();
-  const appName = "Metabase";
-  return getAuthConfiguration(config, dispatch, appName);
+
+  return setupSdkAuth(config, dispatch);
```

Comment:
nit: I wouldn't return anything here since we don't expect the return value.

```suggestion
  setupSdkAuth(config, dispatch);
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/47821/commits/d94cac934b6a6556e6bc05cc97ccc054d03beba1 and fixed the unit test, thanks!